lib/ui/views/sheets/aeBridge_sheet.dart

```diff
@@ -28,14 +26,14 @@ class DEXSheet extends ConsumerStatefulWidget {
   static const String routerPage = '/dex';
 
   @override
-  ConsumerState<DEXSheet> createState() => DEXSheetState();
+  ConsumerState<AEBridgeSheet> createState() => AEBridgeSheetState();
 }
 
-class DEXSheetState extends ConsumerState<DEXSheet> {
-  String? aeSwapUrl;
+class AEBridgeSheetState extends ConsumerState<AEBridgeSheet> {
+  String? aeSwapUrl = 'https://bridge.testnet.archethic.net/bridge';
```

Comment:
On ne devrait pas remettre la remote conf ici ?

Reply:
i did a static page to inform user about bridge access